src/common/test/summarize-profile.js

```diff
@@ -24,18 +24,19 @@ describe('summarize-profile', function () {
 
     let i = 0;
     for (const { category, samples, percentage } of [
-      { category: 'idle', samples: 142, percentage: 0.48464163822525597 },
+      { category: 'wait', samples: 141, percentage: 0.4812286689419795 },
```

Comment:
It's better than no test! It was helpful to be able to eye the diff and ensure that the changes in various categories at least made sense. It would be a slight improvement to express the expected percentages as fractions. Or just remove them.